src/main/java/graphql/schema/GraphQLList.java

```diff
@@ -79,4 +79,10 @@ public TraversalControl accept(TraverserContext<GraphQLType> context, GraphQLTyp
     public List<GraphQLType> getChildren() {
         return Collections.singletonList(wrappedType);
     }
+
+    @Override
+    public String toString() {
+        return GraphQLTypeUtil.getUnwrappedTypeName(this);
```

Comment:
I must say unwrappedTypeName is highly confusing: I would expect for a [Foo] it is Foo, but it is [Foo]